assert/assertions.go

```diff
@@ -1120,6 +1121,24 @@ func NotZero(t TestingT, i interface{}, msgAndArgs ...interface{}) bool {
 	return true
 }
 
+// FileExists asserts that the filename passed represents an existing file.
+//
+//  assert.FileExists(t, "main.go")
+//
+// Returns whether the assertion was successful (true) or calls Fail.
+func FileExists(t TestingT, filename string, msgAndArgs ...interface{}) bool {
+	_, err := os.Lstat(filename)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return Fail(t, fmt.Sprintf("unable to find file %q (%s)", filename, err), msgAndArgs...)
```

Comment:
Why should `err` be printed here?

Reply:
I took a look and I thought to return the error 'cause IMHO the testing is easier.
What do you think @Hanzei?